drivers/hmis/app/graphql/resolvers/validation_errors.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Resolvers
+  class ValidationErrors < Resolvers::Base
+    type [Types::HmisSchema::ValidationError], null: false
+
+    # Resolve HmisErrors and AR errors as a flattened list of HmisErrors
```

Comment:
@eanders I went with implementing an error resolver that can convert ActiveModel Errors into HmisErrors. That way we I didn't to mess with the mutations, which can return a mix of AR errors and/or HMIS errors. The logic for converting AR errors to HMIS errors is encapsulated in the `HmisErrors::Error` class, rather than in the ValidationError GQL type.

Reply:
I like it!